packages/shopify-app-remix/src/boundary/__tests__/error.test.ts

```diff
@@ -0,0 +1,28 @@
+import {shopifyApp} from '../../index';
+import {testConfig} from '../../__tests__/test-helper';
+
+describe('Error boundary', () => {
+  it('returns a string when handling an ErrorResponse', () => {
```

Comment:
What is it about returning a string that gets us the behaviour we want? I think there is something I'm not understanding here. Happy to chat on tuple